README.md

```
@@ -6,31 +6,69 @@ You can read about the technical specifications of the Ootle in the [RFCs](https
-See the dedicated [README](./applications/tari_validator_node/README.md) for installation and running guides.
+The Tari Ootle Wallet Daemon is available on the project’s [releases page](https://github.com/tari-project/tari-ootle/releases).
+Unzip the binaries, then run:
+
+```shell
+tari_ootle_walletd --network igor -b <yourdesiredconfigfolderpath>
+```
+
+This will start a wallet connected to the Igor Testnet. You can view the public Nodes here:
 
-## Running and testing a validator
+- Validator Node: [http://18.217.22.26:12006]](http://18.217.22.26:12006)
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)